docs/verify/simulate-with-qiskit-primitives.mdx

```diff
@@ -71,36 +71,70 @@ estimator = Estimator()
 ### Run and get results
```

Comment:
(making this comment below the actual line)

Generic BaseOperator is not supported in V2 because that's not generally scalable. I suggest that

```
The first step is to define the observables whose expected value you want to compute. Each observable can be any `BaseOperator`, like the operators from [`qiskit.quantum_info`](../api/qiskit/quantum_info).
Among them it is preferable to use [`qiskit.quantum_info.SparsePauliOp`](../api/qiskit/qiskit.quantum_info.SparsePauliOp).
```
changes to
```
The first step is to define the observables whose expected values you want to compute. Each observable can be a Pauli string, a dictionary mapping Pauli strings to real coefficients, or a [`qiskit.quantum_info.SparsePauliOp`](../api/qiskit/qiskit.quantum_info.SparsePauliOp).
In this example, we define a single observable, but in general, you can attach many observables to a single circuit.
```

Reply:
(making this comment below the actual line)

I suggest
```
Next, instantiate an [`qiskit.primitives.Estimator`](../api/qiskit/qiskit.primitives.Estimator).

from qiskit.primitives import Estimator
estimator = Estimator()
```

be changed to

```
Next, instantiate an [`qiskit.primitives.StatevectorEstimator`](../api/qiskit/qiskit.primitives.StatevectorEstimator).

from qiskit.primitives import StatevectorEstimator
estimator = StatevectorEstimator()
```